versions.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+# we will support at most two entries in each of these lists, and both should be in descending order
+supportedDebianSuites=(
+	bookworm
+)
+supportedAlpineVersions=(
+	3.18
+)
+defaultDebianSuite="${supportedDebianSuites[0]}"
+declare -A debianSuites=(
+	[11]='' # https://github.com/docker-library/postgres/issues/582 😬
+)
+defaultAlpineVersion="${supportedAlpineVersions[0]}"
+declare -A alpineVersions=(
+	#[14]='3.16'
+)
+
+gosuVersion='1.16'
+
+cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+versions=( "$@" )
+if [ ${#versions[@]} -eq 0 ]; then
+	versions=( */ )
+	json='{}'
+else
+	json="$(< versions.json)"
+fi
+versions=( "${versions[@]%/}" )
+
+packagesBase='https://raw.githubusercontent.com/redis/redis-hashes/master/README'
+
+declare -A packages=
+_raw_package_list() {
+	curl -fsSL "$packagesBase"
+}
+fetch_package_list() {
+	local -; set +x # make sure running with "set -x" doesn't spam the terminal with the raw package lists
+
+	# normal (GA) releases end up in the "main" component of upstream's repository
+	if [ -z "${packages}" ]; then
+		packages="$(_raw_package_list)"
```

Comment:
This could be just the `curl` directly since the scraped url doesn't change per version.